docs/running-on-kubernetes.md

```diff
@@ -78,7 +78,7 @@ logs and remains in "completed" state in the Kubernetes API until it's eventuall
 Note that in the completed state, the driver pod does *not* use any computational or memory resources.
 
 The driver and executor pod scheduling is handled by Kubernetes. Communication to the Kubernetes API is done via fabric8, and we are
-currently running <code>kubernetes-client</code> version <code>4.1.0</code>. Make sure that when you are making infrastructure additions that you are aware of said version. It is possible to schedule the
+currently running <code>kubernetes-client</code> version <code>4.7.1</code>. Make sure that when you are making infrastructure additions that you are aware of said version. It is possible to schedule the
```

Comment:
Can you link which JIRA changed the version?

Reply:
SPARK-26742(4.1.2)/SPARK-28921(4.4.2)/SPARK-27812(4.6.1)/SPARK-30243(4.6.4)/SPARK-30715(4.7.1)

Reply:
How about just not specifying the version here? or linking to where it's declared in the latest release or something.

Reply:
thanks, @srowen, maybe we don't need to mention fabaric8 version here particularly.